docker-integration-tests/pom.xml

```diff
@@ -34,6 +34,13 @@
     <sbt.project.name>docker-integration-tests</sbt.project.name>
   </properties>
 
+  <repositories>
+    <repository>
+      <id>db2</id>
+      <url>https://app.camunda.com/nexus/content/repositories/public/</url>
```

Comment:
This should be good for now. In parallel, I am working on getting an official ibm maven repo with these drivers, but it's a bureaucratic process.